server/src/test/java/org/opensearch/action/admin/cluster/health/ClusterHealthResponsesTests.java

```diff
@@ -110,9 +110,9 @@ public void testClusterHealth() throws IOException {
         assertThat(clusterHealth.getActiveShardsPercent(), is(allOf(greaterThanOrEqualTo(0.0), lessThanOrEqualTo(100.0))));
```

Comment:
`discovered_master` -> `discovered_cluster_manager` [here](https://github.com/opensearch-project/OpenSearch/blob/737843fcc447c4e9167edc1e24a7ed59c043241c/server/src/test/java/org/opensearch/action/admin/cluster/health/ClusterHealthResponsesTests.java#L166)

Reply:
I will address it in separate PR, and will paste the link later. `discovered_master` is still a existing field in cluster health API, so it will not be renamed directly. I will add additional tests for it.

Reply:
Created new PR https://github.com/opensearch-project/OpenSearch/pull/3432 to rename `discovered_master` in the unit test.